kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/actions/CacheDeleteAction.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+package org.kitodo.mediaserver.core.actions;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import org.kitodo.mediaserver.core.api.IAction;
+import org.kitodo.mediaserver.core.config.FileserverProperties;
+import org.kitodo.mediaserver.core.db.entities.Work;
+import org.kitodo.mediaserver.core.util.FileDeleter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Delete cached derivatives.
+ */
+public class CacheDeleteAction implements IAction {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CacheDeleteAction.class);
+
+    private FileserverProperties fileserverProperties;
+
+    private FileDeleter fileDeleter;
+
+    @Autowired
+    public void setFileserverProperties(FileserverProperties fileserverProperties) {
+        this.fileserverProperties = fileserverProperties;
+    }
+
+    @Autowired
+    public void setFileDeleter(FileDeleter fileDeleter) {
+        this.fileDeleter = fileDeleter;
+    }
+
+    @Override
```

Comment:
It would be nice with some javadoc here

Reply:
You're right. I saw @Override and simply ignored it :-)